.gitignore

```diff
@@ -20,3 +20,4 @@ schematics/Jaccard_index_binary.png
 docs/29_algorithm_validation/ideas.ipynb
 docs/29_algorithm_validation/solution for exercise - metrics to investigate segmentation results.ipynb
 *.png
+*.csv
```

Comment:
Ah, we should not ignore all CSV files. Can you change that to the specific csv files you would like to have ignored?